project.clj

```diff
@@ -2,11 +2,11 @@
   :description "The Fluree Command Line Interface (CLI) is a terminal-based tool that allows users to read and verify Fluree ledger files and consensus logs."
   :url "https://github.com/fluree/fluree.cli"
   :license "SEE LICENSE IN LICENSE"
-  :dependencies [[org.clojure/clojure "1.10.1"]
-                 [org.clojure/data.avl "0.0.19"]
-                 [cheshire "5.8.1"]
-                 [fluree/raft "0.11.1"]
-                 [com.fluree/crypto "0.3.4" :exclusions [org.clojure/clojurescript]]
+  :dependencies [[org.clojure/clojure "1.10.2"]
+                 [org.clojure/data.avl "0.1.0"]
+                 [cheshire "5.10.0"]
+                 [com.fluree/raft "1.0.0-beta1"]
+                 [com.fluree/crypto "0.3.5" :exclusions [org.clojure/clojurescript]]
                  [cli4clj "1.7.6" :exclusions [org.clojure/core.async]]
                  [clj-figlet "0.1.1"]
                  [com.damballa/abracad "0.4.13"]]
```

Comment:
Should we convert this to tools-deps too soon?

Reply:
Not super high priority, spoke about Graal with this as well... just trying to get in an out of the code here though for this patch.